### ISSUE TITLE
Pade two-body Jastrow factor documentation is missing

### ISSUE THREAD

**Issue by shivupa** [QMCPACK/qmcpack#3158]
**Is your feature request related to a problem? Please describe.**
The documentation for the Pade two-body Jastrow factor doesn't exist. The one-body Pade Jastrow factor is documented.
https://qmcpack.readthedocs.io/en/develop/intro_wavefunction.html#two-body-jastrow-functions

It seems that the functional form used is 
`\frac{a*r}{1+b*r} - \frac{a/b}`
rather than eq.11
`\frac{a*r + c*r^2}{1+b*r}`

**Comment by prckent** [comment]
Good point. Have you looked at qmcpack/tests/molecules/Li2_STO_ae and the README there? This has checks for VMC and DMC vs http://dx.doi.org/10.1063/1.465195 (Umrigar's small timestep error DMC algorithm paper). It uses a two body Pade and the published wavefunctions, so any variations in the definitions should be clear.
